content/docs/concepts/components/buildpack-group.md

```diff
@@ -0,0 +1,53 @@
++++
+title="Buildpack Group"
+weight=3
+summary="A buildpack group is a list of specific buildpacks composed together in an order such that the list is suitable for building an application. Because buildpacks are modular and reusable, a buildpack group is what allows you to connect multiple modular buildpacks together."
++++
+
+## What is a buildpack group?
+
+{{< param "summary" >}}
+
+For example, you might have a buildpack that installs Java and a buildpack that uses Maven to build your application. These two buildpacks can be combined into a group to implement higher-level functionality, specifically that the first one will install Java and the second will use Java to run Maven, which is a Java build tool.
+
+Because you can have many buildpack groups in a builder or meta-buildpack and you can reuse buildpacks, you could have a second buidlpack group that reuses the buildpack to provide Java but uses a third buildpack that provides Gradle to build your application. By doing this, you can create additional high-level functionality without having duplication.
+
+## Anatomy of a buildpack group
+
+A [buildpack group][buildpack-group] is a list of buildpack entries defined in the order in which they will run.
+
+A buildpack entry is identified by an id and a version. It may also be marked as optional. While you may have one or more buildpacks in a buildpack group, you may have one or more buildpack groups in a builder or meta-buildpack.
+
+## Detection with buildpack groups
+
+A builder or meta-buildpack may contain multiple buildpack groups. When the lifecycle executes the detection process, it will process each buildpack group it finds in the order that the groups are specified. For each buildpack group, the lifecycle will execute the detect binary of all buildpacks in that group (these can be executed in parallel) and aggregate the results. The lifecycle will select the first builpack group by order where all of the non-optional buildpacks in that group pass detection.
```

Comment:
```suggestion
A builder or meta-buildpack may contain multiple buildpack groups. When the lifecycle executes the detection process, it will process each buildpack group it finds in the order that the groups are specified. For each buildpack group, the lifecycle will execute the detect phase of all buildpacks in that group (these can be executed in parallel) and aggregate the results. The lifecycle will select the first builpack group by order where all of the non-optional buildpacks in that group pass detection.
```